.github/actions/send-teams-notification/action.yml

```diff
@@ -30,7 +30,30 @@ runs:
 
     - name: Compute workflow status
       id: compute-workflow-status
-      uses: martialonline/workflow-status@fe13c6a4716673e224038aa1b02387352fb35e13 # v4
+      shell: bash
+      run: |
+        url="${GITHUB_API_URL}/repos"
+        repo="${GITHUB_REPOSITORY}"
+        run_id="${GITHUB_RUN_ID}"
+        token="${{ github.token }}"
```

Comment:
better move token to `env`?

Reply:
tbh I copy/pasted the original, I can improve indeed